src/System.Private.CoreLib/shared/System/MemoryExtensions.cs

```diff
@@ -1607,5 +1607,113 @@ public static bool Overlaps<T>(this ReadOnlySpan<T> span, ReadOnlySpan<T> other,
                 value, comparer);
             return BinarySearch(span, comparable);
         }
+
+        /// <summary>
+        /// Sorts the elements in the entire <see cref="Span{T}" /> 
+        /// using the <see cref="IComparable" /> implementation of each 
+        /// element of the <see cref= "Span{T}" />
+        /// </summary>
+        /// <param name="span">The <see cref="Span{T}"/> to sort.</param>
+        /// <exception cref = "InvalidOperationException"> 
+        /// One or more elements do not implement the <see cref="IComparable" /> interface.
+        /// </exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Does this really needs AggressiveInlining? Why would JIT not inline a trivial method like this?

 We use AggressiveInlining unless it is absolutely required for achieve good performance. Every use of AggressiveInlining  should be supported by numbers.

Reply:
@jkotas I have removed these, they were simply copied from the former PR.